src/library/Server/Manager/Whm.php

```diff
@@ -129,324 +176,371 @@ public function synchronizeAccount(Server_Account $a)
+        // Log the username change
+        $this->getLog()->info('Changing account ' . $account->getUsername() . ' username');
 
+        // Define the action and parameters for the API request
         $action = 'modifyacct';
-        $var_hash = [
-            'user' => $a->getUsername(),
-            'newuser' => $new,
+        $varHash = [
+            'user' => $account->getUsername(), // Current username
+            'newuser' => $newUsername, // New username
         ];
 
-        $this->_request($action, $var_hash);
+        // Send the request to the WHM API
+        $this->request($action, $varHash);
 
         return true;
     }
 
-    public function changeAccountDomain(Server_Account $a, $new)
+    /**
+     * Changes the domain of an account on the WHM server.
+     *
+     * @param Server_Account $account The account for which to change the domain.
+     * @param string $newDomain The new domain.
+     * @return bool Returns true if the domain was successfully changed.
+     * @throws Server_Exception If an error occurs during the request.
+     */
+    public function changeAccountDomain(Server_Account $account, string $newDomain): bool
     {
-        $this->getLog()->info('Changing account ' . $a->getUsername() . ' domain');
+        // Log the domain change
+        $this->getLog()->info('Changing account ' . $account->getUsername() . ' domain');
 
+        // Define the action and parameters for the API request
         $action = 'modifyacct';
-        $var_hash = [
-            'user' => $a->getUsername(),
-            'domain' => $new,
+        $varHash = [
+            'user' => $account->getUsername(), // Username of the account
+            'domain' => $newDomain, // New domain
         ];
 
-        $this->_request($action, $var_hash);
+        // Send the request to the WHM API
+        $this->request($action, $varHash);
 
         return true;
     }
 
-    public function changeAccountIp(Server_Account $a, $new)
+    /**
+     * Changes the IP of an account on the WHM server.
+     *
+     * @param Server_Account $account The account for which to change the IP.
+     * @param string $newIp The new IP.
+     * @return bool Returns true if the IP was successfully changed.
+     * @throws Server_Exception If an error occurs during the request.
+     */
+    public function changeAccountIp(Server_Account $account, string $newIp): bool
     {
-        $this->getLog()->info('Changing account ' . $a->getUsername() . ' ip');
+        // Log the IP change
+        $this->getLog()->info('Changing account ' . $account->getUsername() . ' ip');
 
+        // Define the action and parameters for the API request
         $action = 'setsiteip';
-        $var_hash = [
-            'domain' => $a->getDomain(),
-            'ip' => $new,
+        $varHash = [
+            'domain' => $account->getDomain(), // Domain of the account
+            'ip' => $newIp, // New IP
```

Comment:
Okay.. Why?
Both variables are descriptively named

Reply:
Whoops, that may have been before variables were renamed. Removing

Reply:
Looks like there are a few occurences of this, apologies for the oversight! Removing them